kalite/distributed/templates/distributed/base.html

```diff
@@ -224,6 +224,9 @@
         </div> {# End wrapper #}
         {# Footer #}
         {% include "distributed/partials/_footer.html" %}
+
+        <script type="text/javascript" src="{% static 'js/tota11y.min.js' %}"></script>
```

Comment:
I would rather just have this automatically come on when DEBUG is True, @aronasorman. That way we can be poking devs in the eye when they make bad accessibility choices!